src/backend/app/projects/project_routes.py

```diff
@@ -1224,17 +1221,7 @@ async def project_dashboard(
     Returns:
         ProjectDashboard: The project dashboard details.
     """
-    data = await project_crud.get_dashboard_detail(db_project, db_organisation, db)
-
-    background_task_id = await project_crud.insert_background_task_into_database(
```

Comment:
We are not saving the submission in s3 cache anymore?
Is fetching the data from central api everytime we need the submissions efficient ?  We should think of the case when there are thousands of submission json.

Reply:
yes we don't need to save the submissions for regular fetch we can do that directly from central, but we might have to consider it when downloading submissions , need to check the performance for that. if it is considerably good then no more s3 if not we not think about it.

Reply:
Later, if the performance doesn't meet expectations, we can create a separate endpoint to save submissions in s3 instead of running background tasks in every endpoint, which can be hit from the frontend whenever someone loads the submission table page.

Reply:
Okay, I am merging this PR for now.